.hgignore

```diff
@@ -1,2 +1,11 @@
+syntax: glob
```

Comment:
I'm using hg-git.
I would have added a new .hgignore, but it was already there.
I suppose I could have added to .gitignore instead.


On Fri, Jun 24, 2016 at 10:53 AM, Dmitry Shachnev <notifications@github.com>
wrote:

> In .hgignore
> <https://github.com/jaraco/keyring/pull/233#discussion_r68435581>:
>
> > @@ -1,2 +1,11 @@
> > +syntax: glob
>
> This repository is using Git, so changing .hgignore doesn't make much
> sense. I don't know why this file is still here, perhaps @jaraco
> <https://github.com/jaraco> is using it via hg-git or something like that.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jaraco/keyring/pull/233/files/e6a057132ecd0027ae115de1465cc6ce0b0c7807#r68435581>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGrjcF80kOKKQvSscwg8v3-i6LmQA4Mks5qPBmCgaJpZM4I9-zW>
> .
>